lib/rubycritic/analysers/smells/flog.rb

```diff
@@ -16,7 +18,13 @@ def initialize(analysed_modules)
       def run
         @analysed_modules.each do |analysed_module|
           add_smells_to(analysed_module)
+          print green "."
         end
+        puts ""
+      end
+
+      def to_s
+        "flog smells"
```

Comment:
How about doing 

>> self.class.name

instead and then put this method into the BaseAnalyzer (see above)?